### ISSUE TITLE
is there any plan to release the training code ？

### ISSUE THREAD

**Issue by valencebond** [Alibaba-MIIL/ASL#1]

**Comment by mrT23** [comment]
Hi valencebond 

Due to commercial limitations, we cannot release at this point the full training code
We provide comprehensive implementation details in the article, and we release our trained models so
that our reported mAP score can be validated

In addition, we provide full implementation of our new loss function
this implementation can serve as a **drop-in replacement** for common losses (cross-entropy, focal loss),
so you can test our proposed loss on basically any repo

Tal

**Comment by valencebond** [comment]
@mrT23  thanks~
